UPGRADE.md

```diff
@@ -3,6 +3,22 @@
 This document serves as a reference to upgrade your current BetterReflection installation if improvements, deprecations
 or backwards compatibility (BC) breakages occur.
 
+## 5.0.0
+
+### BC Breaks
+* Only PHP 8.0 is supported.
+* Adapters don't have `::export()` method anymore because these methods were removed from PHP. These methods has been removed:
```

Comment:
Adapters follow the signature of the parent implementation: unsure if they need to be considered BC breaks 🤔

Let's keep it though - it's a good upgrade consideration 👍